documentation/guides/draft/rust/get-started/xx-discovery-using-cloud-node/README.md

```diff
@@ -0,0 +1,137 @@
+# Discovery using cloud node
+
+## Introduction
+
+In the previous guide we learned how to [create a cloud node](../xx-cloud-node) and connect to it using the TCP transport.
+
+Cloud nodes can be used as proxies to connect devices not exposed by public hostnames or IPs. In this case the cloud node would redirect the traffic from one node to another using the transport connections established by devices.
```

Comment:
We should wordsmith this with something similar to the NAT statement elsewhere. A place to highlight network challenges